core/commands/log_graph.py

```diff
@@ -8,7 +8,7 @@
 
 COMMIT_NODE_CHAR = "●"
 COMMIT_NODE_CHAR_OPTIONS = "●*"
-COMMIT_LINE = re.compile("[%s][ /_\|\-.]*([a-z0-9]{3,})" % COMMIT_NODE_CHAR_OPTIONS)
+COMMIT_LINE = re.compile("^[%s][ /_\|\-.]*([a-z0-9]{3,})" % COMMIT_NODE_CHAR_OPTIONS)
```

Comment:
Won't this make [running an action][1] fail on any commits that don't start with `*`/`●` ? E.g when viewing a commit in a branch, it often starts with `|` such as:

```
●   a573d30 Merge pull request #672 from stoivo/diffstat (4 days ago) <Simon>
|\  
| ● 1d22b92 Settings: New setting `show_diffstat` defaults to true (5 days ago) <Simon>
| ● 56723e7 Enhancement: Show diffstat when showing a commit (5 days ago) <Simon>
|/  
● 1680baa Adds a setting to disable git init prompts (#665) (8 days ago) <Adrian L Lange>
```

[1]: https://github.com/3v1n0/GitSavvy/blob/a06d40d761f3c363db6e92b03bc6935bbe2fdeb8/core/commands/log_graph.py#L196